docs_src/microservices/configuration/CommonEnvironmentVariables.md

```diff
@@ -69,6 +69,9 @@ This environment variable overrides the [`-c/--confdir` command-line option](../
 !!! edgey "EdgeX 3.0"
     The `EDGEX_CONF_DIR` environment variable is replaced by `EDGEX_CONFIG_DIR` in EdgeX 3.0.
 
+!!! edgey "EdgeX 3.0"
```

Comment:
IMO, this `edgey` is not needed here since this is above the environment variable overrides.
That said, the "`-c/--confdir` command-line option" above needs to be "`-cc/--configDir` command-line option" and all instances of `EDGEX_CONF_DIR` need to change to `EDGEX_CONFIG_DIR` except in the `edgey` above.